ui/desktop/electron-app/config/forge.config.js

```diff
@@ -44,7 +44,8 @@ module.exports = {
       name: "@electron-forge/maker-dmg",
       config: {
         name: "boundary-desktop",
-        title: "Boundary Desktop"
+        title: "BoundaryDesktop",
```

Comment:
Removing this title space got me consistent background image in dmg.  @randallmorey , can you give this a try with space? Or is this acceptable?

Reply:
https://github.com/LinusU/node-appdmg/issues/159